scripts/pi-hole/php/footer.php

```diff
@@ -59,6 +59,11 @@
     $coreReleasesUrl = $coreUrl . "/releases";
     $ftlReleasesUrl = $ftlUrl . "/releases";
     $webReleasesUrl = $webUrl . "/releases";
+
+    $dockerhubBaseUrl = "https://hub.docker.com/r/pihole";
+    $dockerRepoUrl = $dockerhubBaseUrl . "/pihole";
+    $dockerTagsUrl = $dockerRepoUrl . "/tags";
+    $dockerTagsFilterUrl = $dockerTagsUrl . "?page=1&name=";
```

Comment:
**Note:**
The `page=1` is not necessary, since the first page is the default page.

Reply:
Yeah! I know that! Just we don't have control on Docker Hub, so I don't know the spec and defined default behavior would be changed or not, simply following the behavior we found on the webpage to have the parameter seems safer to me!